core/amber/src/main/python/core/models/test_RTableExecutor.py

```diff
@@ -0,0 +1,494 @@
+import pandas
+import pytest
+import rpy2.rinterface_lib.embedded
+from core.models import Tuple, Table
+from core.models.RTableExecutor import RTableSourceExecutor, RTableExecutor
+
+
+class TestRTableExecutor:
+    @pytest.fixture
+    def source_executor_empty(self):
+        return """
+        function() {
+            df <- data.frame(
+                col1 = character(),
+                col2 = numeric(),
+                col3 = logical()
+                )
+            return (df)
+        }
+        """
+
+    @pytest.fixture
+    def udf_executor_empty_return(self):
+        return """
+        function(input, port) {
+            return (input)
+        }
```

Comment:
this is echoing input table. can it work if return 
null?

Reply:
Added new test cases for handling null. Returning null works if it is from the Source Operator, but not in the UDF Operator since it is impossible to convert null back to Arrow